libethereum/Client.h

```diff
@@ -286,7 +286,7 @@ class Client: public ClientBase, protected Worker
 	/// @warning May be called from any thread.
 	void onBadBlock(Exception& _ex) const;
 
-	BlockChain m_bc;						///< Maintains block database and owns the seal engine.
+	std::unique_ptr<BlockChain> m_bc;		///< Maintains block database and owns the seal engine.
```

Comment:
I think we cannot make that a pointer - I would think that too many objects hold references to structures inside this.